libs/utils/utils.js

```diff
@@ -880,6 +897,7 @@ export async function loadArea(area = document) {
 
   // Load everything that can be deferred until after all blocks load.
   await loadDeferred(area, areaBlocks, config);
+  scrollToHashedElement();
```

Comment:
I'm a bit concerned about the timing of this, but I'll leave it up to the milo core team to have the final call on this

Reply:
Sorry @mokimo , this will not work, as load eventListener does not get trigger here and Safari is not working as expected. Will have to add it in script.js. Looking for other ways to handle this problem. scrollToHashedElement() is fine, it is working as expected but figuring out how to handle load eventListener without calling it in script.js